tests/test_expression.py

```diff
@@ -220,7 +220,13 @@ def get_context_data(
 
         self.assertEqual(
             rendered.strip(),
-            "<!-- _RENDERED SimpleComponent_743413,a1bc3e -->\n                <div>lorem ipsum dolor</div>\n                <div>True</div>\n                <div>[{'a': 1}, {'a': 2}]</div>\n                <div>{'a': 3}</div>",  # noqa E501
+            (
+                "<!-- _RENDERED SimpleComponent_743413,a1bc3f,, -->\n"
```

Comment:
Why is this comment ending with two commas?

Reply:
I treat the text after the `_RENDERED` as a row of CSS ([as described here](https://github.com/EmilStenstrom/django-components/pull/861#discussion_r1898891705)). So the two commas means that there's no data for those fields, which means that the rendered component doesn't define any JS nor any CSS variables.